lib/include/openpiv/core/util.h

```diff
@@ -17,6 +17,14 @@
 // openpiv
 #include "core/exception_builder.h"
 
+// define POSIX-style endianness constants for MSVC
```

Comment:
out of interest, what requires these defines to be hard coded?

Reply:
`pnm_image_loader.cpp` uses these macros which are only defined in unix. I had to manually add them to support windows and I assumed that windows always runs little-endian. See the example provided here that does the same:
https://en.cppreference.com/w/cpp/types/endian